src/data/linux-quiz.ts

```diff
@@ -1246,6 +1246,16 @@ const linuxQuiz = [
     Explanation:
       "The ls -X command is used to list the file extensions in alphabetical order.",
     Link: "https://www.freecodecamp.org/news/the-linux-commands-handbook/"
+  },
+  {
+    Question: "In Linux, how many operations does the CAT command do?",
```

Comment:
It looks like you can do more than 4 operations with the CAT command based on these articles here
https://www.geeksforgeeks.org/cat-command-in-linux-with-examples/
https://phoenixnap.com/kb/linux-cat-command

```suggestion
    Question: "In Linux, what are four operations that can be done with the cat command?",
```